lib/DismissiblePrompt/stories/DismissiblePromptStory.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import DismissiblePrompt from '..';
+import StoryItem from '../../../stories/styleguide/StoryItem';
+
+storiesOf('Components', module).add('DismissiblePrompt', () => (
+  <div>
+    <StoryItem
+      title="DismissiblePrompt"
+      description="A small prompt that can be dissmissed"
+    >
+      <DismissiblePrompt>hello there!</DismissiblePrompt>
```

Comment:
does this require an `onDismiss` prop? I can't see a default one

Reply:
keen eye! i'll add in